src/engine/effects/message.h

```diff
@@ -97,10 +95,10 @@ struct EffectsRequest {
         } RemoveEffectChain;
         struct {
             EffectStatesMapArray* pEffectStatesMapArray;
-            const ChannelHandle* pChannelHandle;
+            ChannelHandle channelHandle;
```

Comment:
I was expecting that ChannelHandle with its custom constructor is not usable in this union. Removing the memset seem to work, though.

Reply:
That seems really brittle and easy to break when refactoring.

Reply:
agree -- for now we think it's just better to state that we don't guarantee initialization and callers are responsible for it.  We can clean this up with the followup rewrite using variant or whatever